gitops/snapshot.go

```diff
@@ -118,6 +118,18 @@ const (
 
 	//AppStudioIntegrationStatusFinished is the reason that's set when the AppStudio tests finish.
 	AppStudioIntegrationStatusFinished = "Finished"
+
+	//IntegrationTestStatusQueuedGithub is the status reported to github when integration test is in a queue
+	IntegrationTestStatusQueuedGithub = "queued"
```

Comment:
Is there a guidance why queued and not pending, something in GH docs? How these values were decided?

Reply:
These statuses will be used to report to Github. I found there is a `pending`,  `queued` is set by GitHub itself.

Reply:
I know that they are used to report to GH, but but why they have those specific values.
Is there a GH doc about statuses which can be used?

Reply:
sorry missed it